ncm-metaconfig/src/main/metaconfig/libvirtd/pan/schema.pan

```diff
@@ -111,3 +111,77 @@ type service_sysconfig_libvirtd = {
     'sdl_audiodriver' ? string with match(SELF, '^(pulse)$')
     'libvirtd_nofiles_limit' ? long (1..)
 };
+
+
+type type_kvmvm_network = {
+    @{linux or OVS bridge name required by the network interface}
+    'bridge' : string
+    @{mac address required by the VM}
+    'mac' : type_hwaddr
+    @{only required to use a Open vSwitch bridge}
+    'type' ? string with match (SELF, '^openvswitch$')
+} = dict();
+
+type type_kvmvm_rbd = {
+    @{name of the block device available from the storage pool.
+     it should incldue the relative path to the storage pool, as example:
+     "one/disk1.vda"}
+    'name' : string
+    @{list of Ceph monitors}
+    'ceph_hosts' : type_fqdn[]
+} = dict();
+
+@documentation{
+Parameters required to use a Ceph storage backend
+}
+type type_kvmvm_ceph_disk = {
+    @{uuid of the libvird secret generated from Ceph secret.xml file
+     more info: http://docs.ceph.com/docs/hammer/rbd/libvirt/
```

Comment:
use http://docs.ceph.com/docs/master/rbd/libvirt/ instead of the hammer specific one